src/main/java/alice/Alice.java

```diff
@@ -0,0 +1,85 @@
+package alice;
+
+import alice.task.TaskList;
+
+/**
+ * This class is the representation of the agent being created. this class is
+ * immutable and any method calls from this class should not have any side
+ * effects. Message returned to the user should be stored in the agent and
+ * output of it should be handled elsewhere.
+ */
+
+public class Alice {
+	public static final String GREETING = "Hello!";
```

Comment:
Maybe you could consider adding an empty line between the class and the string variable to improve clarity? However, if you feel that it is fine this way, let it be.